src/rust/lib/graph-editor/src/component/visualization.rs

```diff
@@ -1,29 +1,36 @@
 //! This module defines the visualization widgets and related functionality.
 //!
-//! The overall architecture of visualizations consists of three parts:
+//! The overall architecture of visualizations consists of four parts:
 //!
 //! 1. The `DataRenderer` trait provides the functionality to render the actual visualization view
 //!    that implements `display::Object`. It is provided with data and itself provides frp streams
 //!    of its output if there is some, for example, if it acts as a widget.
 //!
 //! 2. The `Visualization` is a struct that wraps the `DataRenderer` and implements the generic
-//!    tasks that are the same for all visualisations. That is, interfacing with the other UI
+//!    tasks that are the same for all visualizations. That is, interfacing with the other UI
 //!    elements, the visualization registry, as well as propagating frp messages.
 //!
-//! 3. The `Container` wraps the `Visualisation` and provides the UI elements that facilitate
-//!    user interactions. For example, selecting a visualisation or connecting it to nodes in the
+//! 3. The `Class` is a struct that instantiates `Visualization` structs and provides data about
+//!    the visualization even before they are instantiated like input datatype and name.
```

Comment:
Just a minor change request to make it more obvious "that instantiates" -> "that can instantiate multiple"